shiny/templates/app-templates/dashboard/app-core.py

```diff
@@ -5,94 +5,101 @@
 
 from shiny import App, Inputs, Outputs, Session, reactive, render, ui
 
-sns.set_theme(style="white")
-df = pd.read_csv(Path(__file__).parent / "penguins.csv", na_values="NA")
 species = ["Adelie", "Gentoo", "Chinstrap"]
 
+species_images = {
```

Comment:
I kind of wanted to make the dashboard pretty minimal, so I'm not sure about adding images. My experience is that images in value boxes is a bit rare. Also we probably don't want to link out to web sites that might change.

Reply:
I agree, especially for a template.